src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -251,11 +276,16 @@
     <!-- We call MakePri.exe to create common resources.pri file only if this test project has its own resources. -->
     <Exec Command="$(_MakePriCommand)" StandardOutputImportance="Low" StdErrEncoding="Unicode" Condition="'$(TestProjectNeedsModifiedPriFile)' == 'true'" />
 
-    <Copy Condition="'$(TestProjectNeedsModifiedPriFile)' != 'true'"
+    <Copy Condition="'$(TestProjectNeedsModifiedPriFile)' != 'true' AND Exists('$(_CommonPriFile)')"
           SourceFiles="$(_CommonPriFile)"
           DestinationFiles="$(TestPath)\resources.pri"
           SkipUnchangedFiles="true" />
 
+    <Copy Condition="'$(TestProjectNeedsModifiedPriFile)' != 'true' AND !Exists('$(_CommonPriFile)')"
```

Comment:
The Runner\resources.pri file should always exist as it comes from the Runner's package and that is the base PRI that we use to merge from to create the CommonPriFile. The reason why I added this is in the scenario where a repo doesn't have any resources embedded into their projects and the external dependencies doesn't have resources either (this case is not likely to happen, but just thinking of the worst scenario) the list of resw files will be empty so MakePri will not create the CommonPriFile because it had no inputs. 

So I did this to protect the build, so that when this happens and there is no CommonPriFile, we use the base file which comes from the runner itself, the build succeeds and the tests wont fail.